charset_normalizer/md.py

```diff
@@ -21,6 +21,25 @@
 )
 
 
+SUSPICIOUS = {
```

Comment:
Thats the exact opposite. I would name this constant something along the line of `COMMON_SAFE_ASCII_CHARACTERS`.